Library/Formula/freeswitch.rb

```diff
@@ -138,6 +140,7 @@ class Freeswitch < Formula
   #------------------------ End sound file resources --------------------------
 
   def install
+    ENV.deparallelize
```

Comment:
Can you add a link to the upstream bug submission and explanation of why this is needed in a comment in the formula file. Thanks!